rpc/legacyrpc/methods.go

```diff
@@ -17,6 +17,9 @@ import (
 	"sync"
 	"time"
 
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
```

Comment:
there should not be any grpc imports in the json-rpc server implementation, this server has its own unique error codes